Dockerfile

```diff
@@ -47,6 +47,7 @@ COPY --chown=gi-bill-data-service:gi-bill-data-service . .
 USER gi-bill-data-service
 RUN gem install bundler --no-document -v ${BUNDLER_VERSION}
 RUN bundle install --binstubs="${BUNDLE_APP_CONFIG}/bin" $bundler_opts && find ${BUNDLE_APP_CONFIG}/cache -type f -name \*.gem -delete
+RUN bundle exec whenever --update-crontab --set environment=test
```

Comment:
I believe this should be the production environment, yes?

Reply:
I updated it taking out the env

Reply:
@rileyanderson  can you take a look at this.

Reply:
@PJart  see comments above.

Reply:
I updated the dockerfile to set it to production, by just setting it to bundle exec whenever --update-crontab, it sets the crontab to run without a specified env. The schedule commands run the commands to run the job.

Reply:
just to test it I can set the crontab in staging as well, we just need to test that the cron job can be run with the changes I made to schedule.rb